docs/guides/src/main/server/logging.adoc

```diff
@@ -7,102 +7,116 @@ title="Configuring logging"
 summary="Learn how to configure Logging"
 includedOptions="log-*">
 
-Keycloak uses the jboss logmanager logging framework. The high-level overview for the available log handlers is shown below:
+Keycloak uses the jboss logmanager logging framework. The following is a high-level overview for the available log handlers:
 
```

Comment:
I'd suggest rewriting to "Keycloak uses the JBoss Logging framework...."

Reply:
Thanks, @abstractj. Corrected.